test/integration/game_test.js

```diff
@@ -1,13 +1,108 @@
 const chai = require('chai');
 const expect = chai.expect;
 
+const fs = require('fs');
+
+
 const app = require('../../src/app');
 const gameState = require('../../src/league');
 
+describe('JSON files for league app test suite', function (){
+  //TODO: test for input file to be present
```

Comment:
Do you have a way of tracking this TODO? Or is it at risk of dropping off the radar?